java/server/src/org/openqa/selenium/grid/node/remote/RemoteNode.java

```diff
@@ -200,6 +201,11 @@ public HealthCheck getHealthCheck() {
     return healthCheck;
   }
 
+  @Override
+  public void drain() {
```

Comment:
What if the local node this points to has the drain cancelled somehow?

Reply:
I have it in my notes that we discussed this back in October, and we decided that wouldn't be a design goal for this first iteration. I don't mind building that in, but only if you consider it a requirement for a first pass. Just let me know what you think.